scripts/git_clang-format_hook.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+set -Eeuo pipefail
+
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+# This is the continuous delivery build script executed after a git
+# extract by the Jenkins build process located at the following URL:
+# https://builds.apache.org/view/M-R/view/Qpid/job/Qpid-proton-c/
+#
+
+# https://gist.github.com/alexeagle/c8ed91b14a407342d9a8e112b5ac7dab
+
+readonly help=$(cat <<- EOF
+  This repository requires you to install the git clang-format command.
+  One-time setup step (on Fedora Linux):
+     $ dnf install git-clang-format
```

Comment:
Maybe include the url to git-clang-format as well just in case there isn't a git-clang-format package in their distro.